README.md

```diff
@@ -1 +1,183 @@
+
+<div align="center">
+  <h3><b>GamerPower Application</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[GamerPower\] ](#-gamerpower-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Demonstration Video ](#-demonstration-video-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [GamerPower] <a name="about-project"></a>
+
+**[GamerPower application]** is the ultimate tool for gamers! With GamerPower, you can easily find, track, and collect the best giveaways in gaming. Whether you're looking for free games, beta keys, free in-game loot, or currency for your favorite games, you won't miss out on a single freebie with GamerPower by your side!
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.gamerpower.com/api/giveaways">Open GamePower API</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Global state using Redux]**
+- **[API driven application]**
+- **[Single Page Application]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Demonstration Video -->
+
+## 🚀 Demonstration Video <a name="live-demo"></a>
+
+- [Demonstration Video](https://www.loom.com/share/0ce4f760e4c446c9b89603bbb8851f31)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Install Node.JS
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/AlexisJW/react-capstone-project.git
+```
+
+### Install
+
+Install this project with:
+
+Install dependencies of this project with:
+
+```sh
+  cd react-capstone-project
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+-
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm test
+```
+
+### Deployment
+
+> - [Deployment](https://gamepower.onrender.com/home)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] [REQUIRED] Hello @AlexisJW , it looks like our deployment link is broken. Let's ensure that the deployment is successful and that the URL attached here works 👍🏽

Reply:
Thanks Mr. @Whoistolu, I'll make the changes.